apphub/instance_segmentation/solov2/solov2_tf.py

```diff
@@ -533,7 +523,7 @@ def get_estimator(data_dir=None,
     model = fe.build(model_fn=lambda: solov2(input_shape=(im_size, im_size, 3)),
                      optimizer_fn=lambda: tf.optimizers.SGD(learning_rate=init_lr, momentum=0.9))
     network = fe.Network(ops=[
-        Normalize(inputs="image", outputs="image", mean=(0.485, 0.456, 0.406), std=(0.229, 0.224, 0.225)),
+        Normalize(inputs="image", outputs="image", mean=(123.675, 116.28, 103.53), std=(58.395, 57.12, 57.375)),
```

Comment:
@TortoiseHam current implementation, expects the mean and std to be in actual values (eg:(123, 125, 124), (12456, 12546, 12868)) instead of 0-1. This is to avoid the implicit /255 or /65025 we have to do based on the input image values.

Reply:
does that mean there's no default if user doesn't want to manually provide these values?

Reply:
So the default value is None, we we compute it automatically inside the normalize function.